### DISCUSSION TITLE
RiMusic UI

### DISCUSSION THREAD

**Discussion by OrangeZXZ** [fast4x/RiMusic#70] [Ideas]
Hi everyone!
This Discussion was made so that everyone can say their opinion on the RiMusic UI and make concepts.

Feel free to share any ideas!
If you like an idea add an ↑ (It would be easier to see witch idea peoples like the most)

ONLY RULE : Don't post bugs here because there is already "Issues"

**Comment by fast4x** [comment]
I like it

**Comment by ikanakova** [comment]
As I wrote in the issue #65 I think it would be cool if the app was skinnable:

- in a minimal look there would be an album image (or equalizer, if applicable), song and artist info (with links to go to album, artist and without "now playing...") + previous, play/pause, next buttons
- (optionally) underneath there would be another bar with smaller buttons e.g. favourite, etc. and three dots with other things like start radio, etc.
- (optionally) either a bar with info about the song now playing (without links) and next song/s - maybe that one could be to the right of the album or just a little bar with next song at the bottom.
- choice of font and icons (outline/file/sharp)
- choice album image or equalizer (if applicable) or lyrics

**Comment by fast4x** [comment]
Yes I agree with @ikanakova.

Buttons in the bottom bar of version 0.6.7 i don’t like much, better move them.

I wish RiMusic wasn’t a copy of other forks.

**Reply by ikanakova** [reply]
I agree to make the UI original. At the same time, I think it should be based on ViMusic and extend it.

**Reply by OrangeZXZ** [reply]
I agree buttons in the bottom was a bad idea but the next song text i find it better there I can try to remake a concept without those ( I think i have great ideas )

Oh yeah also it is hard to open the bottom bar because the text in it is clickable

**Comment by fast4x** [comment]
Which download icon do you prefer between the first and the third?
![Screenshot_2023-11-04-10-07-57-134_com.microsoft.emmx.jpg](https://github.com/fast4x/RiMusic/assets/1484476/e5122b3e-edac-43e3-8b7f-66641be91167)

**Reply by OrangeZXZ** [reply]
I would say the 3 one is the most simple to understand because the cloud can mean other things

**Reply by fast4x** [reply]
I agree, also because the first one I find very generic

**Reply by ikanakova** [reply]
I also like the 3rd one the best, but the 1st one makes me feel more like people could understand that it's not a classic download, but just a download from the cloud to the cache.

**Reply by dainvincible1** [reply]
3rd one looks good

**Reply by fast4x** [reply]
> I also like the 3rd one the best, but the 1st one makes me feel more like people could understand that it's not a classic download, but just a download from the cloud to the cache.

The detail you added is interesting, Let’s think about it after next release.

**Reply by ikanakova** [reply]
I think the app uses icons Filled. Although in the case of <img src="https://unpkg.com/ionicons@7.1.0/dist/svg/download-outline.svg" width="20"> I like the outline better, but I would fatten the lines to make it more massive than the original outline.

**Reply by fast4x** [reply]
i changed all outline icons to filled
and added new icon stats

**Reply by ikanakova** [reply]
I'm downloading all the svg icons from Ionicons right now. I'm putting them in an .assets/icons file so that a) we can always access them even if ionicons deletes them, and b) so that we can add more of our own custom icons, modified from ionicons. So far, I have a modified addqueue.

**Reply by ikanakova** [reply]
Application icons are just in the app/src/main/res/drawable folder, right @fast4x?

I created a new addqueue (so far filled, sharp and outline I will add).

locked and unlocked should either be replaced by assets/ionicons/lock-open.svg and lock-closed.svg or I will create whatever you want based on them.

Are play_arrow, play_stop, play_skip_previous, play_skip_next, play_pause used in the application? Because they are sharp variants of play, play-skip-back, etc.

I created a pull request with a copy of all ionicons and my created icons (stats etc.)

**Reply by fast4x** [reply]
> Application icons are just in the app/src/main/res/drawable folder, right @fast4x?
> 
> I created a new addqueue (so far filled, sharp and outline I will add).
> 
> locked and unlocked should either be replaced by assets/ionicons/lock-open.svg and lock-closed.svg or I will create whatever you want based on them.
> 
> Are play_arrow, play_stop, play_skip_previous, play_skip_next, play_pause used in the application? Because they are sharp variants of play, play-skip-back, etc.
> 
> I created a pull request with a copy of all ionicons and my created icons (stats etc.)

Actually some are in drawable and other in anydensity.

I have changed lock/unlock icons with filled version.

**Reply by fast4x** [reply]
> Are play_arrow, play_stop, play_skip_previous, play_skip_next, play_pause used in the application? Because they are sharp variants of play, play-skip-back, etc.

I'm not sure, I'll check it.

**Comment by senolfndk** [comment]
<img width="224" alt="frame_4_4x" src="https://github.com/fast4x/RiMusic/assets/52818836/59740147-e5a9-45fc-b7d4-426d07d3bb61">

**Reply by senolfndk** [reply]
![image_processing20211113-17959-8ypok9](https://github.com/fast4x/RiMusic/assets/52818836/2b8d0b8c-e2ad-4c9c-8283-e91a901b9db6)

**Reply by OrangeZXZ** [reply]
I like the simple look i can try to add this in my next UI Concept for RiMusic

**Reply by fast4x** [reply]
> ![image_processing20211113-17959-8ypok9](https://github.com/fast4x/RiMusic/assets/52818836/2b8d0b8c-e2ad-4c9c-8283-e91a901b9db6)
> 

Great, would be a new theme.

**Reply by ikanakova** [reply]
Nice, especially the simple buttons scheme. I just think the progress bar should be classic so it's not a [copy](https://dribbble.com/shots/16851412-Music-Player-Material-You).

**Comment by dainvincible1** [comment]
> <img alt="frame_4_4x" width="224" src="https://user-images.githubusercontent.com/52818836/280478200-59740147-e5a9-45fc-b7d4-426d07d3bb61.png">

looks real goood, very nice

**Comment by fast4x** [comment]
Work in progress :)

![Screenshot_20231104_215028](https://github.com/fast4x/RiMusic/assets/1484476/8ae96b4d-9e0a-4451-8969-af038a082be2)

**Comment by NEVARLeVrai** [comment]
> Work in progress :)
> 
> ![Screenshot_20231104_215028](https://github.com/fast4x/RiMusic/assets/1484476/8ae96b4d-9e0a-4451-8969-af038a082be2)
> 

U have ri music in french lol

**Reply by ikanakova** [reply]
Great! But I think the filled version <img src="https://unpkg.com/ionicons@7.1.0/dist/svg/settings.svg" width="20"> looks better than the outline <img src="https://unpkg.com/ionicons@7.1.0/dist/svg/settings-outline.svg" width="20"> Especially when the pause icon etc are also filled.

**Reply by OrangeZXZ** [reply]
> Great! But I think the filled version <img src="https://unpkg.com/ionicons@7.1.0/dist/svg/settings.svg" width="20"> looks better than the outline <img src="https://unpkg.com/ionicons@7.1.0/dist/svg/settings-outline.svg" width="20"> Especially when the pause icon etc are also filled.

maybe do like the search button (add background circle)

**Reply by ikanakova** [reply]
> maybe do like the search button (add background circle)

I meant it like this: <img src="https://github.com/fast4x/RiMusic/assets/38765251/c22f3ff3-3faf-4479-a9aa-6d1ab838e139" width="20"> (on a black background). Search button is also filled one. I meant that all icons should be either filled or outline.

**Reply by fast4x** [reply]
> Great! But I think the filled version <img src="https://unpkg.com/ionicons@7.1.0/dist/svg/settings.svg" width="20"> looks better than the outline <img src="https://unpkg.com/ionicons@7.1.0/dist/svg/settings-outline.svg" width="20"> Especially when the pause icon etc are also filled.

Just changed... 😋

![Screenshot_2023-11-06-20-02-50-986_it.fast4x.rimusic.jpg](https://github.com/fast4x/RiMusic/assets/1484476/b8a87d5a-03f5-4351-a2ff-82dcf6cc7d59)

**Comment by NEVARLeVrai** [comment]
> > <img alt="frame_4_4x" width="224" src="https://user-images.githubusercontent.com/52818836/280478200-59740147-e5a9-45fc-b7d4-426d07d3bb61.png">
> 
> looks real goood, very nice

It look nice, it corresponds to Android 13

**Reply by OrangeZXZ** [reply]
Don't forget to reply in the threads! 😁

**Comment by fast4x** [comment]
Would you like RiMusic to be browsable on horizontal pages?

Example

Quick picks ->  Swipe  -> Songs -> etc...

Likes this 
https://developer.android.com/jetpack/compose/layouts/pager?hl=it

**Reply by OrangeZXZ** [reply]
On my redesign concept v2 i was considering adding it horizontaly

**Reply by senolfndk** [reply]
yes

**Reply by ikanakova** [reply]
I think horizontal scrolling (right and left) should be left as it is now for scrolling between songs in Tips etc. and vertical scrolling (up and down) (VerticalPager) should be for scrolling between Quick Picks, Albums etc.

**Comment by OrangeZXZ** [comment]
Brand NEW UI Concept right here: https://github.com/fast4x/RiMusic/issues/89#issue-1981647344

Don't hesitate to give ideas or opinions!

**Comment by senolfndk** [comment]


**Reply by OrangeZXZ** [reply]
It looks very similar to what I've made and i think that it would be nice that i add the album name to the top

**Reply by senolfndk** [reply]
might be a good idea

**Comment by ikanakova** [comment]
If you need a new icon for something, I think it would be great to open a feature request and I'll be happy to create it so all the icons look the same.

Likewise if you suggest modifying one of them.

**Reply by OrangeZXZ** [reply]
I think this icon should be changed
![SmartSelect_20231125_165335.jpg](https://github.com/fast4x/RiMusic/assets/104271365/cfb145da-6974-4af3-b4d0-32976c9eed43)

Maybe use this icon with a clock or something like this:
![SmartSelect_20231125_165427.jpg](https://github.com/fast4x/RiMusic/assets/104271365/06daa0e9-6c1d-48f8-9ad1-e08aee201191)

**Comment by senolfndk** [comment]
![Screenshot_20231130-161645_Oto Music](https://github.com/fast4x/RiMusic/assets/52818836/370e5b8b-751d-4696-8f5d-caa0456c9d66)

**Reply by OrangeZXZ** [reply]
The time bar on the third image could be an idea

**Reply by fast4x** [reply]
Yes, i agree

**Comment by Snyret** [comment]
Main Section:
![Screenshot_20231202_024736.jpg](https://github.com/fast4x/RiMusic/assets/139090371/3754a5ef-e02b-4484-b4b7-a8ee428bb0d3)

Are you referring to the search icon on the right?

**Reply by Snyret** [reply]
Um, yeah the search icon would look more natural if it was in the same style as the others in my opinion.

Can't believe I'm responding to an edit of my comment hahaha.

**Reply by Snyret** [reply]
![Screenshot_20231202_024715.jpg](https://github.com/fast4x/RiMusic/assets/139090371/23ddafa2-d18c-49ce-8452-a175a8863ff5)

Also this section that you removed is just a little unaligned.

**Reply by fast4x** [reply]
Sorry

**Comment by senolfndk** [comment]


**Reply by fast4x** [reply]
This logo for RiMusic app or?

**Reply by senolfndk** [reply]
![Ekran görüntüsü 2023-12-14 164154](https://github.com/fast4x/RiMusic/assets/52818836/1b2e002b-9439-4361-b2ef-c28ec5ca3a3e)

**Reply by senolfndk** [reply]
yes

**Reply by fast4x** [reply]
It’s cute but very similar to the icon of a Linux media application...

**Reply by senolfndk** [reply]
hmm

**Reply by senolfndk** [reply]
![Ekran görüntüsü 2023-12-14 164654](https://github.com/fast4x/RiMusic/assets/52818836/6d35e98a-0cb7-4da1-9b0f-436ea4269024)

**Reply by fast4x** [reply]
also this

**Reply by senolfndk** [reply]
If there is someone who is an expert in this field, she can make a material logo.

**Reply by senolfndk** [reply]
Because the default logo is very mediocre, there could be a more beautiful and magnificent logo instead

**Reply by fast4x** [reply]
Do not say "mediocre" but different from the rest. Respect the current colors and themes available. Of course you can always improve thanks to ideas like yours.

**Reply by senolfndk** [reply]
I'm sorry buddy

**Reply by senolfndk** [reply]
I respect the work you do 🤗

**Reply by senolfndk** [reply]
I'm just trying to offer suggestions and ideas. 🙂

**Reply by fast4x** [reply]
Don't worry, no problem, each of us has our own respectable idea, this is the beauty of open source

**Reply by OrangeZXZ** [reply]
Personally i use custom icons from LuxRed
![SmartSelect_20231214_163903_One UI Home.jpg](https://github.com/fast4x/RiMusic/assets/104271365/336f58d9-6ebf-45eb-a4cd-8a56be67fe64)

I sended an request to make the RiMusic Icon that way I can use it

**Comment by senolfndk** [comment]


**Reply by fast4x** [reply]
Nice

**Reply by OrangeZXZ** [reply]
I like the time bar could it be added?

**Reply by fast4x** [reply]
Of course, it falls into the idea of adding various types of timeline

**Reply by fast4x** [reply]
Implemented new timelines, Pin Bar like this and Bodied Bar :)
Available from 0.6.13

**Comment by OrangeZXZ** [comment]
IM IMPRESSED!

The UI in the 0.6.13 version is stunning
Also the audio visualizer dosen't crash anymore!
Edit: now it freezes but runs smoother

![Screenshot_20231216_000554.jpg](https://github.com/fast4x/RiMusic/assets/104271365/6db87ca4-4eec-451f-8801-651b726aee77)

**Comment by OrangeZXZ** [comment]
There is an app named Innertune that has an incredible UI but it lacks features

![Screenshot_20240127_233623_InnerTune.jpg](https://github.com/fast4x/RiMusic/assets/104271365/bddb03e3-e149-4724-bb15-e315ae2fda61)

![Screenshot_20240127_233628_InnerTune.jpg](https://github.com/fast4x/RiMusic/assets/104271365/62a0e938-192f-48e6-b954-38f6487476df)

![Screenshot_20240127_233639_InnerTune.jpg](https://github.com/fast4x/RiMusic/assets/104271365/6b13f5d1-aee4-4603-9f89-89c682234efc)

![Screenshot_20240127_233648_InnerTune.jpg](https://github.com/fast4x/RiMusic/assets/104271365/8ba10311-2609-4a2c-80a9-a74fc5dacb10)

I think im going to do an Issue with things that could be added from the innertune (while respecting the RiMusic UI)

Also there is a but that makes the play button black on the dinamic theme

**Comment by chamudha** [comment]
Can't add pure black theme with dynamic theme ![PicsArt_01-28-11.16.53.jpg](https://github.com/fast4x/RiMusic/assets/134184550/97b9c565-26da-4827-8ce5-4ebcab05f8d7)

**Reply by fast4x** [reply]
Theme "dynamic" can't be pure black

**Reply by chamudha** [reply]
ok. Can't add a meterial disigned the option for themes ?![PicsArt_01-28-10.26.58.jpg](https://github.com/fast4x/RiMusic/assets/134184550/ea0ca60d-fb0c-4435-b36f-6e60646f60b0)

**Reply by OrangeZXZ** [reply]
> ok. Can't add a meterial disigned the option for themes ?![PicsArt_01-28-10.26.58.jpg](https://github.com/fast4x/RiMusic/assets/134184550/ea0ca60d-fb0c-4435-b36f-6e60646f60b0)
> 
> 

I dont think he can for now since he needs to change a lot of the code

**Comment by 6XChen** [comment]
Can I request to use this icon in RiMusic? 🙃
![Dark RiMusic.png](https://github.com/fast4x/RiMusic/assets/104673054/e492842c-7cca-48cf-a35c-885e09635e39)

**Reply by fast4x** [reply]
Of course you can, but the text would be a duplicate...

**Reply by 6XChen** [reply]
What would a duplicate look like???

**Reply by fast4x** [reply]
> What would a duplicate look like???

Imagine your icon as an app to launch, the word RiMusic would be in and out of the icon

![Screenshot_2024-01-28-17-37-46-935_com.miui.home-edit.jpg](https://github.com/fast4x/RiMusic/assets/1484476/c11dcec3-7f89-42bd-af1d-0e3ca385cfbd)

**Reply by 6XChen** [reply]
Or like this??
![Dark RiMusic 2.png](https://github.com/fast4x/RiMusic/assets/104673054/0d7e43ed-79db-4f67-80cb-b1e1cd1a6c5b)

**Reply by ikanakova** [reply]
I don't like this. It's too much color for me. Plus the white on green is to make the icon monochrome as well.

**Reply by 6XChen** [reply]
> I don't like this. It's too much color for me. Plus the white on green is to make the icon monochrome as well.

What colour do you like??

**Reply by ikanakova** [reply]
> > I don't like this. It's too much color for me. Plus the white on green is to make the icon monochrome as well.
> 
> What colour do you like??

I like it the way it is. White on green.

**Reply by 6XChen** [reply]
> > > I don't like this. It's too much color for me. Plus the white on green is to make the icon monochrome as well.
> > 
> > What colour do you like??
> 
> I like it the way it is. White on green.

But I need dark mode icon 😢, it looks really cool to show off to friends 🫠

**Reply by OrangeZXZ** [reply]
> But I need dark mode icon 😢, it looks really cool to show off to friends 🫠

If you have OneUI you can add custom icons with "ThemePark"

![SmartSelect_20240128_180150_One UI Home.jpg](https://github.com/fast4x/RiMusic/assets/104271365/1f26c321-a2fa-47b6-8cef-bf37ef3b348e)

**Reply by ikanakova** [reply]
> > > > I don't like this. It's too much color for me. Plus the white on green is to make the icon monochrome as well.
> > > 
> > > 
> > > What colour do you like??
> > 
> > 
> > I like it the way it is. White on green.
> 
> But I need dark mode icon 😢, it looks really cool to show off to friends 🫠

The monochrome icon should work by automatically apply the app icon color based on your wallpaper or selected color palette (Works on Android version 13 or higher). I tried to set it up that way, but I'm not a programmer, so maybe maybe it doesn't work. That's why it's white in the foreground.

**Reply by 6XChen** [reply]
> > > > > I don't like this. It's too much color for me. Plus the white on green is to make the icon monochrome as well.
> > > > 
> > > > 
> > > > What colour do you like??
> > > 
> > > 
> > > I like it the way it is. White on green.
> > 
> > But I need dark mode icon 😢, it looks really cool to show off to friends 🫠
> 
> The monochrome icon should work by automatically apply the app icon color based on your wallpaper or selected color palette (Works on Android version 13 or higher). I tried to set it up that way, but I'm not a programmer, so maybe maybe it doesn't work. That's why it's white in the foreground.

Wouldn't it be more interesting if it had a matching colour to this design?? 🙃![Screenshot_20240129_071316_GitHub.png](https://github.com/fast4x/RiMusic/assets/104673054/38a7e12a-627a-4584-84cb-8d838c54ac21)

**Reply by OrangeZXZ** [reply]
> Wouldn't it be more interesting if it had a matching colour to this design?? 🙃![Screenshot_20240129_071316_GitHub.png](https://github.com/fast4x/RiMusic/assets/104673054/38a7e12a-627a-4584-84cb-8d838c54ac21)

It works for the dark theme but currently apps, sites, etc focus more on the light theme and the green colored icon works better and fades better with other apps if it was black it would pop too mutch

**Reply by 6XChen** [reply]
> > Wouldn't it be more interesting if it had a matching colour to this design?? 🙃![Screenshot_20240129_071316_GitHub.png](https://github.com/fast4x/RiMusic/assets/104673054/38a7e12a-627a-4584-84cb-8d838c54ac21)
> 
> It works for the dark theme but currently apps, sites, etc focus more on the light theme and the green colored icon works better and fades better with other apps if it was black it would pop too mutch

Btw, can we add a feature to change the app icon?? 🫠

**Reply by ikanakova** [reply]
> > > Wouldn't it be more interesting if it had a matching colour to this design?? 🙃![Screenshot_20240129_071316_GitHub.png](https://private-user-images.githubusercontent.com/104673054/300304174-38a7e12a-627a-4584-84cb-8d838c54ac21.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY1MDQyODgsIm5iZiI6MTcwNjUwMzk4OCwicGF0aCI6Ii8xMDQ2NzMwNTQvMzAwMzA0MTc0LTM4YTdlMTJhLTYyN2EtNDU4NC04NGNiLThkODM4YzU0YWMyMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyOVQwNDUzMDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZGFiZDc2YTY1MzA2MzNiMWVhNGFhYWU2NWVlNmQ3MTljOGVhNzlhMThmNjU1MGNjMmQzM2Q1M2ViNzk2NTE3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.cTHaR453cNoFmC-n5Cl9MiucSip5__s_yge7Kj684tg)
> > 
> > 
> > It works for the dark theme but currently apps, sites, etc focus more on the light theme and the green colored icon works better and fades better with other apps if it was black it would pop too mutch
> 
> Btw, can we add a feature to change the app icon?? 🫠

I prefer a light theme and I don't like the black icon on it. It would be better if you could choose the icon color. For the sake of theme customization, monochrome icons tend to be white in the foreground because you don't know what specific theme color the user will have, but it certainly won't be white but it could easily be green.

**Reply by OrangeZXZ** [reply]
> Btw, can we add a feature to change the app icon?? 🫠

This can be doable it would let peoples use the ViMusic icon but I find it not usefull and very gimmicky

**Reply by ikanakova** [reply]
> > Btw, can we add a feature to change the app icon?? 🫠
> 
> This can be doable it would let peoples use the ViMusic icon but I find it not usefull and very gimmicky

I don't think this is a good idea, the app is supposed to have 1 icon. But it would be rather cool if, as in the case of Fossify apps (formerly Simple Mobile Tools), you could change the icon background color at will - the icon is also white in the foreground, but you can choose if the background is black, green, purple, etc.
<p float="left">
<img src="https://github.com/fast4x/RiMusic/assets/38765251/cd23f1d6-8e29-4fd1-87a3-7161e390dbf4" width="250">
<img src="https://github.com/fast4x/RiMusic/assets/38765251/7bd8d913-e573-485d-88ac-0b6a6ad9f162" width="250">
</p>

**Comment by chamudha** [comment]
Showing big icon in notification area ![Screenshot_2024-02-01-09-54-05-317_lockscreen.jpg](https://github.com/fast4x/RiMusic/assets/134184550/53e25653-c821-4374-bb4f-adeaf712deb2)

**Comment by OrangeZXZ** [comment]
![SmartSelect_20240311_072420.jpg](https://github.com/fast4x/RiMusic/assets/104271365/5dcab69d-15fb-4e80-934d-ef315e845616)

I was wondering what meant this eye icon in the newest update?

**Reply by fast4x** [reply]
Visualizza il totale delle canzoni incluse le mai ascoltate.

**Reply by OrangeZXZ** [reply]
> Visualizza il totale delle canzoni incluse le mai ascoltate.

what do you mean songs not listened?

**Reply by fast4x** [reply]
Example, when you click on a song in an album or playlist, all songs present are added to the db as a reference.

**Comment by MillionsToOne** [comment]
Is there a way to rearrange songs in playlist? I feel like the UI is so cluttered I can't find a common feature that every music client should have.

**Reply by fast4x** [reply]
It is not a question of messy interface but different interface from the rest, uncommon...

In library playlist set order to position, unlock rearrange and you can move order of songs.

Look

![Screenshot_2024-03-29-19-03-50-119_it.fast4x.rimusic.jpg](https://github.com/fast4x/RiMusic/assets/1484476/8a9d1bbe-06df-4e57-9508-3592fb12b226)

**Reply by MillionsToOne** [reply]
It was much better in ViMusic where the features were limited, resulting in clean UI. Having million buttons makes things complicated. It feels more of a thing Apple would do. Making things complicated.

**Reply by fast4x** [reply]
These are personal preferences, you can change in ViMusic mode from the UI settings. I will be happy to welcome your ideas, create tickets for the changes you would like.

**Reply by fast4x** [reply]
> It was much better in ViMusic where the features were limited, resulting in clean UI. Having million buttons makes things complicated. It feels more of a thing Apple would do. Making things complicated. 

Apple? uuuhhhhh never!

**Reply by MillionsToOne** [reply]
> > It was much better in ViMusic where the features were limited, resulting in clean UI. Having million buttons makes things complicated. It feels more of a thing Apple would do. Making things complicated. 
> 
> Apple? uuuhhhhh never!

Yeah, instead of having a simple toggle, you have to do 3 steps before achieving a result you want.

**Reply by MillionsToOne** [reply]
> These are personal preferences, you can change in ViMusic mode from the UI settings. I will be happy to welcome your ideas, create tickets for the changes you would like.

The ViMusic mode changes nothing since it still has all that extra clutter I don't need.

**Reply by senolfndk** [reply]
you can try this https://github.com/25huizengek1/ViTune/releases/tag/v1.0.0

**Reply by fast4x** [reply]
> > These are personal preferences, you can change in ViMusic mode from the UI settings. I will be happy to welcome your ideas, create tickets for the changes you would like.
> 
> The ViMusic mode changes nothing since it still has all that extra clutter I don't need. 

Friend...

**Reply by fast4x** [reply]
> > These are personal preferences, you can change in ViMusic mode from the UI settings. I will be happy to welcome your ideas, create tickets for the changes you would like.
> 
> The ViMusic mode changes nothing since it still has all that extra clutter I don't need. 

Not use RiMusic, change to another player.

**Comment by astro-ray** [comment]
I want to ask two things

1. Is this intentional 
![Screenshot_20240405_222248.jpg](https://github.com/fast4x/RiMusic/assets/69363775/6d2d907c-e337-489b-ac66-035084a211fc)

(Some colour seem to overflow outside the rounded corner border)

2. The contrast seems a bit off for the icons on the top
![Screenshot_20240405_223741.jpg](https://github.com/fast4x/RiMusic/assets/69363775/2ac41b37-ccd9-4043-a9f1-5195a3e6b7a5)

(Both in dynamic and Modern black colour theme)

**Reply by OrangeZXZ** [reply]
Rounded borders are pretty new maybe in the next update is patched pls consider making an issue if the new update dosent add what you asked now

**Reply by fast4x** [reply]
1 is a progress bar in miniplayer, can disable by settings

2 is a gradient background in player, can disable by settings

**Reply by fast4x** [reply]
I guess I didn’t quite understand

**Reply by astro-ray** [reply]
> I guess I didn’t quite understand

I am sorry, I didn't quite explain it right.

![Screenshot_20240405_223741.jpg](https://github.com/fast4x/RiMusic/assets/69363775/b8b12f82-9a18-40f5-aced-2276f675d25d)

As you can see in this case, the visibility of these icons are not very good (or maybe it's just me?).

Also, I didn't realize the gradient could be disabled, thanks for letting me know.

**Reply by fast4x** [reply]
for gradient try enable and disable again

**Comment by senolfndk** [comment]
![0a3932aab7eda4d94df3b533195b46fc](https://github.com/fast4x/RiMusic/assets/52818836/0b031040-08c8-4c6d-9998-7549c7197a5e)
![1b8e6de69a37c035a25104550b5d837a](https://github.com/fast4x/RiMusic/assets/52818836/585c9816-1f33-4d33-9462-c906a8ca4f9d)
![1d2636892fd55c8612bc4e8b1c80c754](https://github.com/fast4x/RiMusic/assets/52818836/283ac28d-31ba-44d3-8f4c-39e9bb9f730d)
![3bb798ea09767c0bfb74f354ea9d7a34](https://github.com/fast4x/RiMusic/assets/52818836/1ed30e45-9541-4fc6-b569-ca5fc2186ff3)
![6a6266fdeb57d6de70711f25505f8ee9](https://github.com/fast4x/RiMusic/assets/52818836/b1d13e52-e2e1-4bea-8f0e-1751159c1030)
![10b2e3a5a12bdb4b6d7c253fb00b88af](https://github.com/fast4x/RiMusic/assets/52818836/04a2ab55-5262-4f98-b594-5d83fb9e944d)
![46d54932a6f6b081ff1a17f192ecff65](https://github.com/fast4x/RiMusic/assets/52818836/c1a61d1c-7c90-4b84-a20b-5d7d16ea3347)
![553498f474189aefb3f712a9e290bf38](https://github.com/fast4x/RiMusic/assets/52818836/435438f9-2383-478b-b050-a2d02c54abd5)
![869303576c04131903d2f528456d5392](https://github.com/fast4x/RiMusic/assets/52818836/cd4e341f-873d-4a66-85ed-3841847e11dd)
![f4407bafe543e66be908ef7a0c0d8afd](https://github.com/fast4x/RiMusic/assets/52818836/b16c3941-3b89-433c-8033-456321675ae6)
![original-7e9bd9a69a60c212b294f21dacd0526e](https://github.com/fast4x/RiMusic/assets/52818836/18c0eaac-3d7d-43f8-9309-ba1cd1e66a7a)
![original-9ae6b43d13818ade695b3c213fd39b5b](https://github.com/fast4x/RiMusic/assets/52818836/915d058b-4a8f-4b58-8f64-c03509894f6b)
![original-561a11106653185c809db54fefe021da](https://github.com/fast4x/RiMusic/assets/52818836/8e36057f-1aff-49b2-8f04-c53f00ea9dfa)
![original-d209b8a491030ce60b3463ac158e27ad](https://github.com/fast4x/RiMusic/assets/52818836/89a5a7fb-7409-4af4-94c8-119d8e3fe66f)
![original-ff1d6e266c4aeb5f512070dced555fc0](https://github.com/fast4x/RiMusic/assets/52818836/e17b5969-a605-43cb-90ef-29a0f8d58cc4)

**Reply by senolfndk** [reply]
https://dribbble.com/

**Reply by OrangeZXZ** [reply]
I like the first UI i think RiMusic should have more floaty and rounded UI

**Comment by senolfndk** [comment]
![df73655df4b20635060027fa55af328f](https://github.com/fast4x/RiMusic/assets/52818836/f96b569a-e2b3-4a96-a5a2-b690ba5fb2ba)
![original-0a830fb54cd1489da212e79f825d6b2f](https://github.com/fast4x/RiMusic/assets/52818836/510420f7-9451-427b-80d2-953f09548377)
![original-4aa0aad109ecd91d4f2e54c194304c91](https://github.com/fast4x/RiMusic/assets/52818836/8822ebf3-b863-4ee8-9c2f-dc049c601c85)
![original-7f2de0e52bc9a4d8c15d1561431bb689](https://github.com/fast4x/RiMusic/assets/52818836/5579de2d-7dcb-48b0-8624-fdfddef25e20)
![original-9feaaeba3922052a72dca9d4841decf8](https://github.com/fast4x/RiMusic/assets/52818836/84927283-c792-4ef7-bba9-a49faf540c3f)
![original-88ea44429fca87e095fd0ef953a7a978](https://github.com/fast4x/RiMusic/assets/52818836/833c991e-dae5-4d3d-8569-90d229cb6bc4)
![original-97e77928914cf86343631744c0e5ba2d (1)](https://github.com/fast4x/RiMusic/assets/52818836/f6c77235-3aa2-4394-8c3d-7e5aefece8a3)
![original-97e77928914cf86343631744c0e5ba2d](https://github.com/fast4x/RiMusic/assets/52818836/4f3565d1-d1a0-4e93-8c7f-b891f8e50eca)
![original-76913de1956d97e7e0141257b79c1a72](https://github.com/fast4x/RiMusic/assets/52818836/9fe598b9-5668-4d89-8af5-4df3232357a8)
![original-a81757746ea458a2440f7303b5333d98](https://github.com/fast4x/RiMusic/assets/52818836/340e5d12-47f3-4af3-a343-3bd508c114bc)
![original-ae60f7a0d26a700486346c4d40daaaec](https://github.com/fast4x/RiMusic/assets/52818836/f5a05a97-7cea-4da7-aede-c1bf8a082d63)
![original-c32f11cf0b614cd77642ab1f213727b3](https://github.com/fast4x/RiMusic/assets/52818836/8c84cf8e-8812-4f80-881d-e1dda8b5637b)
![original-d153daf8525a31e1703a934716217ecb](https://github.com/fast4x/RiMusic/assets/52818836/27a1d8e6-91ea-4289-9235-bddc2ba44330)

**Comment by chamudha** [comment]
Do you can combine Ri music UI with Auxio, Simhoney or ZenMusic Ui in github repos ?? Ri Music Ui isn't attractive now.
I think it would be good if the ui of Ri music changed.(This is just a suggestion, please consider it if possible)

**Reply by fast4x** [reply]
I think they are too similar because they have a starting point very close to InnerTune. RiMusic is not attractive? It is your thinking based on your ideas.

**Reply by chamudha** [reply]
I just suggested, the decision is yours, I think would be better if there was a channge in the player UI.Sorry if you ware upset.

**Reply by fast4x** [reply]
No, I’m not angry, I just want to tell you that RiMusic doesn’t have to look like an app, it has to point to having distinctive signs. So I ask you, if you have different ideas write as well, the important thing is not to think about making it similar to other apps.

**Comment by OrangeZXZ** [comment]
![Screenshot_20240518_102100.jpg](https://github.com/fast4x/RiMusic/assets/104271365/ec025641-22f3-4a80-929f-9dc1e3d7490b)

The new UI is splendid! Great Work @fast4x and all the peoples that suggested it!

It removes the non visible buttons problem and I like it because it feels more like One UI now THX!!!

**Reply by fast4x** [reply]
Thanks a lot @OrangeZXZ and all other
users
